HidPkg/UsbHidDxe/UsbHidDxe.c

```diff
@@ -26,7 +26,12 @@
 
 #include <IndustryStandard/Usb.h>
 
-#define CLASS_HID  3
+#define CLASS_HID      3
+#define SUBCLASS_BOOT  1
+
+// Refer to USB HID 1.11, section 7.2.6.
+#define BOOT_PROTOCOL    0
+#define REPORT_PROTOCOL  1

```

Comment:
I know these are also defined in UsbKbHidDxe and UsbMouseHid. Do you think it is time to put it somewhere common given it is defined in the USB HID spec?

Reply:
No objection, seems like they would go in MdePkg\Include\IndustryStandard\Usb.h, which currently has some UsbMass class-specific stuff. Would you like me to raise a PR against basecore for this?

Reply:
In any case, if this is added to basecore or someplace else, I propose raising that as a separate change.

Reply:
> In any case, if this is added to basecore or someplace else, I propose raising that as a separate change.

Yes, that's fine and all the uses would need to be updated.